### PR TITLE
PR: MakeMake - GalleryText

### PR DESCRIPTION
**Pull request by ibanzajoe**
1. Issue #8 - GalleryText

This is a redo of the pull request because previous pull request had a different yarn.lock.

**NOTE: Please title PR: {projectName} - {componentName}**

Components included in this PR are:

1. {issueNumber} - {componentName} - With hover state
1. {issueNumber} - {anotherComponentName} - With color and logo variant

NOTE Ideally only one component per PR, but sometimes it makes sense to submit two together if they are linked (like a list with a listItem for example).

## Notes

Anything we need to know, perhaps a note about missing assets, or a hover state etc...

### CODE REVIEW

File: src/components/CR8 - GalleryText/GalleryText.vue [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+            el.style.opacity = 0
+        },
+        enter (el, done) {
+            el.style.opacity = 1
+        },
+        leave (el) {
+            setTimeout(() => {
+                el.style.opacity = 0
+            }, 100)
+        }
```
Comment (drewbaker) [inline src/components/CR8 - GalleryText/GalleryText.vue]: What are these doing?

**Review by drewbaker — CHANGES_REQUESTED** [review]
You can still see the text through the images. 

Any now the hover state of the slideshow doesn't work consistently (mouse over the elastic logo and see). I suspect you need to use @mouseover.native="" for the events perhaps.

I think at this point it would be easier if I just finished this off.

**Review by drewbaker — CHANGES_REQUESTED** [review]
See screenshot. I think you broke z-index somewhere along the line.

The slideshow looks way better now, but there is a black frame at the end before it loops for some reason?

Screen Shot 2020-07-15 at 3 40 10 PM

**Comment by ibanzajoe** [comment]
@drewbaker I forgot to add -1 to the imagesLength on the interval.  But there is a problem with the db.json where the last object in "images" does not have a sourceURL.  So that is where you are getting the black screen.

> See screenshot. I think you broke z-index somewhere along the line.
> 
> The slideshow looks way better now, but there is a black frame at the end before it loops for some reason?
> 
> Screen Shot 2020-07-15 at 3 40 10 PM

**Comment by drewbaker** [comment]
Yeah that is missing a URL by design, it's meant to catch errors like this! :) 

But I think it's fine for this actually, as the API will not export an image with a missing URL, so you can ignore the black screen note.

Let me know when you got that z-index thing fixed and I will merge.

**Comment by ibanzajoe** [comment]
@drewbaker i am actually not seeing the z-index issue.  Can you specify?  The text is NOT supposed to be over the images?  Only the logo?

**Comment by drewbaker** [comment]
Actually, you are right. Text is supposed to be over image. My bad.

**Comment by ibanzajoe** [comment]
@drewbaker cool.  you should still put -1 inside the set interval: if (this.activeImage < this.imagesLength **-1**)  INSIDE the method: start ()

**Pull request merged**